statchatexpress/app.js

```diff
@@ -70,8 +89,29 @@ app.post("/users/login", async (req, res) => {
     if (user.Password !== Password) {
       return res.status(400).json({ message: "Incorrect password" });
     }
+    req.session.userId = user.dataValues.id;
+    req.session.save(function () {
+      res.json({ message: "Login successful" });
+    });
+  } catch (err) {
+    res.status(500).json({ message: err.message });
+  }
+});
+
+app.post("/users/profile", async (req, res) => {
+  try {
+    const { gamertag, platform } = req.body;
+    const userId = req.session.userId;
+    const user = await User.findByPk(userId);
+    if (user) {
+      user.Gamertag = gamertag;
+      user.Platform = platform;
+      await user.save();
 
```

Comment:
This code probably only works for your original implementation - could you confirm?